scripteditor.js

```diff
@@ -1,105 +1,294 @@
   
-  ssRates.getRange('B4').setValue(ZYX);
-  ssRates.getRange('B5').setValue(YXW);
-  ssRates.getRange('B6').setValue(XWV);
-
-  //VTC wallet balances
-  //Add more as needed with different variable names
+  var myCoinsObj = {};
+  var myCoinsCount = myCoins.length;
+  for (var i = 0; i < myCoinsCount; i++) {
+    var c = i+2;
+    var n = 0;
+    while (coins[n]['id'] !== myCoins[i]) {
+      n++;
+    }
+    
+    myCoinsObj[coins[n]['id']] = coins[n];
+        
+    ssRates.getRange('A'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['id']);
+    ssRates.getRange('B'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['name']);
+    ssRates.getRange('C'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['symbol']);
+    ssRates.getRange('D'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['rank']);
+    ssRates.getRange('E'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['price_usd']);
+    ssRates.getRange('F'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['price_btc']);
+    ssRates.getRange('G'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['24h_volume_usd']);
+    ssRates.getRange('H'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['market_cap_usd']);
+    ssRates.getRange('I'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['available_supply']);
+    ssRates.getRange('J'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['total_supply']);
+    ssRates.getRange('K'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['max_supply']);
+    ssRates.getRange('L'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['percent_change_1h']);
+    ssRates.getRange('M'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['percent_change_24h']);
+    ssRates.getRange('N'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['percent_change_7d']);
+    ssRates.getRange('O'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['last_updated']);
+    if (targetCurrency !== 'usd') {
+      ssRates.getRange('P'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['price_' + targetCurrency]);
+      ssRates.getRange('Q'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['24h_volume_' + targetCurrency]);
+      ssRates.getRange('R'+(c).toString()).setValue(myCoinsObj[myCoins[i]]['market_cap_' + targetCurrency]);
+  
+  //  /\     /\     /\     /\     /\     /\
+  // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    };
+  }
 
-  var VtcWallet = getVtcBalance("yourAddressHere");
+  //
+  // =================================
+  //
+  //   WALLET BALANCE CONFIGURATION
+  //
+  // =================================
 
-  //Change the value in getRange() to match the cell in spreadsheet
-  //Change the value in setValue() to match the variable above
+  //
+  // ===== BCH Wallet Balances =======
+  // Add more as needed with different variable names
+  // Change the value in getRange() to match the cell in spreadsheet
+  // Change the value in setValue() to match the wallet variable used (e.g. bchWallet)
+  // =================================
 
-  ssRates.getRange('E3').setValue(VtcWallet);
```

Comment:
VTC wallet points to Rates sheet instead of Wallet

Reply:
👍